capstone-frontend/src/components/AssignmentCard.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+
+import CommentCard from './CommentCard';
+
+import '../styles/Groups.css';
+
+
+class AssignmentCard extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      comments: []
+    }
+  }
+  onComment = (e) => {
+    e.preventDefault();
+    let data = {
+      "assignmentID": this.props.assignment.id,
+      "text": e.target[0].value,
+      "userID": window.localStorage.getItem("userID"),
+      "whenCreated": (new Date()).toUTCString()
+    };
+    fetch("/api/comments", {method: "post", body: JSON.stringify(data)})
+        .then(this.fetchComments())
```

Comment:
This is a nice POST followed by GET paradigm you're following here!

Reply:
Thanks!